modules/compression/src/index.js

```diff
@@ -7,3 +9,19 @@ export {default as LZ4InflateTransform} from './lib/lz4/lz4-inflate-transform';
 // Move to WIP due to enormous dependency size
 // export {default as ZstdDeflateTransform} from './lib/zstd/zstd-deflate-transform';
 // export {default as ZstdInflateTransform} from './lib/zstd/zstd-inflate-transform';
+
+// __VERSION__ is injected by babel-plugin-version-inline
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+export const ZlibWorker = {
+  id: 'zlib',
+  name: 'zlib',
+  version: VERSION,
+  options: {
+    zlib: {
+      localWorkerUrl: 'modules/compression/dist/zlib.worker.js',
+      workerUrl: `https://unpkg.com/@loaders.gl/compression@${VERSION}/dist/zlip.worker.js`
```

Comment:
Do we need two URLs? Loaders only have one

Reply:
Actually, we don't need any, can now autogenerate both of these.